yvalenta/parallel_search.rb

```diff
@@ -0,0 +1,44 @@
+class SearchVowels
+  def initialize
+    @time ||= { start: 0, finish: 0 }
+  end
+
+  def open_file
+    f = File.open(File.dirname(__FILE__) + '/../text.txt')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.